### PR TITLE
Add doc 'Setting up a new project'

### PR DESCRIPTION
**Pull request by anne-edwards**
#### Description
Adding the doc "Setting up a new project"
#### Tests
n/a
#### Documentation
n/d
#### Primary reviewers
@ElleEss @samc-improbable

### CODE REVIEW

File: docs/content/set-up-new-project.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+To use the SpatialOS GDK for Unity in a new project, you need to set up your project manifest, and then set up world initialization. Your new project must have the [same structure as a SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure).
+
+### Set up your project manifest
+Add the following dependencies to the `packages` manifest located inside `workers/unity/Packages/manifest.json`:
+```
+{
+  "dependencies": {
+    "com.improbable.gdk.core": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.core",
+    "com.improbable.gdk.legacy": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.legacy",
```
Comment (zeroZshadow) [inline docs/content/set-up-new-project.md]: This no longer exists and should now be `com.improbable.gdk.buildSystem`

Comment (jessicafalk) [reply docs/content/set-up-new-project.md]: ah, I fixed it in the other doc, but forgot about it here

**Comment by jessicafalk** [comment]
@zeroZshadow this doc is not about the connection flow, but only the very basic setup. The workers doc looks into this a bit. Please read the google docs, if you want to comment on them.

File: docs/content/set-up-new-project.md (hunk 1 above)
Comment (anne-edwards) [reply docs/content/set-up-new-project.md]: @zeroZshadow which bit exactly should be "com.improbable.gdk.buildSystem"?

**Pull request merged**